resources/db-schema.cql

```diff
@@ -0,0 +1,144 @@
+CREATE TABLE forecast_names (
```

Comment:
We should probably consider using something like Joplin for this: https://github.com/juxt/joplin